man/systemd.network.xml

```diff
@@ -2214,6 +2292,17 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
           addresses. See <varname>NetLabel=</varname> in [Address] section for more details.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>NFTSet=</varname></term>
+        <listitem>
+          <para>This applies the NFT set for the network configuration received with DHCP, like
+          <varname>NFTSet=</varname> in [Address] section applies it to static configuration. See
+          <varname>NFTSet=</varname> in [Address] section for more details. For <literal>address</literal> or
+          <literal>prefix</literal> source types, the type of the element used in the NFT filter must be
+          <literal>ipv4_addr</literal>.</para>
```

Comment:
Still, I cannot follow the last sentence. When `address` or `prefix`, the NFT type will be automatically chosen as `ipv4_addr`? Maybe, you want to say `the type of the element used in the NFT filter will be <literal>ipv4_addr</literal>`, that is `must be` -> `will be` ?

Reply:
`networkd` is not creating any NFT rules, tables or sets, they must be prepared elsewhere in advance (perhaps this should be also highlighted). `networkd` only inserts elements to the sets and then the NFT rules must be compatible to the `networkd` configuration or the insertion will fail. The types expected by `networkd` are chosen by the source type and if it's `address` or `prefix`, also whether the address/prefix is ipv4 or ipv6 matters since the NFT rules must then be different. I think `must be` shows this expectation better (I'm not a native English speaker though), but perhaps the wording could be improved.

It might be possible for `networkd` to also create the missing rules, but I'm not sure of the implications. For example, loading the NFT rules later with `nft -f nftables.conf` by a service could fail if the rules don't flush any previous definitions.

Reply:
Added a clarification

Reply:
Thank you for the clarification.